### PR TITLE
[12.0][MIG] account balance line

### PR DESCRIPTION
**Pull request by davidStudio73**
Standard migration, change version and readme structure

### CODE REVIEW

File: account_balance_line/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#. If you're an accounting adviser, enable developer mode and go to
+   Accounting > Adviser > Journal items.
+#. Or open an account or a partner, and click on Action > Journal items.
+#. You will see a column named "Balance" with the balance of the line.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline account_balance_line/readme/USAGE.rst]: Remove this part, as it's now auto-generated

Comment (davidStudio73) [reply account_balance_line/readme/USAGE.rst]: Done

**Review by pedrobaeza — APPROVED** [review]

File: account_balance_line/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2010-2014 Camptocamp - Vincent Renaville
+# Copyright 2016 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Balance on journal items',
+    'summary': 'Display balance totals in move line view',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting',
+    'website': 'http://www.tecnativa.com,',
```
Comment (nikul-serpentcs) [inline account_balance_line/__manifest__.py]: @davidStudio73 Should be `https://github.com/OCA/account-financial-tools`

File: account_balance_line/views/account_move_line_view.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="account_move_line_balance_custom">
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account.view_move_line_tree"/>
+        <field name="arch" type="xml">
+            <field name="credit" position="after" >
+                <field name="balance" sum="Total Balance" invisible="0"/>
```
Comment (nikul-serpentcs) [inline account_balance_line/views/account_move_line_view.xml]: @davidStudio73 `invisible="0"`  Is need ??

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: account_balance_line/__manifest__.py (hunk 2 above)
Comment (davidStudio73) [reply account_balance_line/__manifest__.py]: Changed, thanks for the tip.

File: account_balance_line/views/account_move_line_view.xml (hunk 3 above)
Comment (davidStudio73) [reply account_balance_line/views/account_move_line_view.xml]: no, It's not necessary, removed.

**Review by CasVissers-360ERP — APPROVED** [review]
Functional test, LGTM.

**Pull request merged**